web/src/i18n/ru_RU.js

```diff
@@ -0,0 +1,113 @@
+export default {
+    lang: {
+        choose: 'Выбор языка/Language',
```

Comment:
Please do not translate language display name. 
The language selection should be keep its original translation for user select in any language profile.